andela/denethor-ah-frontend
# Set up redux

#### This pull request
Sets up redux

#### Tasks completed
- Set up redux, redux-thunk and react-redux
- Set up redux testing environment and templates
- Wire up the Stats component to use the store
- Minor bug fixes (repo badges, image/css imports)

#### How to test
- Clone the repo
- `checkout` branch `chore/163478751/setup-redux`
- Add dependencies with `npm i`
- Run server with `npm run dev-server`
- Check out redux-connected UI on a browser
- Run test suite `npm test`
